components/fieldgroups/experience-event/experienceevent-media-collection-downloaded.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-media-collection-downloaded",
```

Comment:
Can you merge the below 3 field groups to 1 ?
media-collection
media-collection-downloaded
media-reporting

You can just move "xdm:mediaReporting" and "xdm:mediaEvents" objects to the media-collection field group.

Reply:
Done. I created a fieldgroup of mediaAnalytics which contains mediaCollection, mediaReporting and mediaDownloadedEvents.